R/render-plot.R

```diff
@@ -37,6 +37,7 @@ renderPlot <- function(expr, width='auto', height='auto', res=72, ...,
   } else {
     installExprFunction(expr, "func", env, quoted)
   }
+  func <- wrapFunctionLabel(func, "renderPlot", ..stacktraceon = TRUE)
```

Comment:
I added it to installExprFunction. As long as functions are using that with default args they should be OK. I think I broke `[Shiny: Reactive]` labels in RStudio debugger though (do we even need those anymore?). cc @jmcphers